### DISCUSSION TITLE
Loopback Support for Azure Functions

### DISCUSSION THREAD

**Discussion by faiztherocker** [loopbackio/loopback-next#7184] [Q&A]
I have been looking out ways in which Azure Functions can be deployed using Loopback, Any source of documentation will be helpful.

**Comment by raymondfeng** [comment]
Do you want to check out https://github.com/raymondfeng/loopback4-example-serverless?

**Comment by srikanthadibhatla** [comment]
@raymondfeng do you have a working example for azure functions ? https://github.com/raymondfeng/loopback4-example-serverless - this seems to be geared for IBM cloud functions
